src/main/java/Duke/gui/Main.java

```diff
@@ -0,0 +1,73 @@
+package duke.gui;
+
+import java.io.IOException;
+
+import duke.Duke;
+import javafx.application.Application;
```

Comment:
I'm really nitpicking here but perhaps adding a line break between `import duke.Duke` and `import javafx.application.Application` would adhere better to the coding standards?
```suggestion
import duke.Duke;

import javafx.application.Application;
```